03/app.js

```diff
@@ -42,11 +42,23 @@ books.getAuthor = function(isbn) {
 }
 
 books.getTitle = function(isbn, lang) {
+    
+     const bookTitle = this[isbn]['title'][lang];
```

Comment:
Trzeba sprawdzić czy wszystkie kolejne elementy występuję inaczej będzie błąd.
Tak jak miało to wcześniej tj. `if(typeof this[isbn] !== 'undefined') {` a potem `if(typeof this[isbn]['title'] == 'undefined') {` itd.